README.md

```diff
@@ -169,71 +241,26 @@ class PartyControllerTest < ActionController::TestCase
 end
 ```
 
-##### RSpec
-
-It is also possible to use `with_variant` and all the individual setup and
-assertions steps in RSpec tests. Here is an example of a Capybara feature file:
-
-```ruby
-# spec/features/ab_testing_spec.rb
-feature "Viewing a page with an A/B test" do
-  include GovukAbTesting::RspecHelpers
-
-  scenario "viewing the B version of the page" do
-    with_variant your_ab_test_name: 'B' do
-      visit root_path
-
-      expect(page).to have_breadcrumbs
-      expect(page).to have_beta_label
-    end
-  end
-end
-```
-
-And here is an RSpec controller test:
-
-```ruby
-# spec/controllers/some_controller_spec.rb
-describe SomeController, type :controller do
-  include GovukAbTesting::RspecHelpers
+## API documentation
 
-  # RSpec doesn't render views for controller specs by default
-  render_views
+See [RubyDoc](http://www.rubydoc.info/gems/govuk_ab_testing) for documentation including all of the assertions for tests.
 
-  it "should render the B version of the page" do
-    with_variant your_ab_test_name: 'B' do
-      get :index
-    end
-  end
-end
-```
-
-As with the `minitest` version, you can also pass in the following options to
-`with_variant`:
+To run a Yard server locally to preview documentation, run:
 
-- `assert_meta_tag: false`
-- `dimension: <number>`
+    $ bundle exec yard server --reload
 
-### Running the test suite
+## Running the test suite for the gem
 
 `bundle exec rake`
```

Comment:
Does this need to be under a heading? Especially a level 2 heading? I wonder if it can be moved to the top of the acceptance testing section somewhere before getting into the specific different testing methods.